app/javascript/dashboard/dashboard.jsx

```diff
@@ -4,16 +4,23 @@ import displayClick from '../../assets/images/display_click.png';
 import displayTime from '../../assets/images/display_time.png';
 import displayComment from '../../assets/images/display_comment.png';
 
-
 export class Dashboard extends Component {
   constructor(props) {
     super(props);
     this.state = {};
   }
 
   render() {
-    /* eslint-disable */ const { visits, visit, comments, time, articles, avgArticles, words, avgWords } = this.props;
-console.log(avgArticles)
+    /* eslint-disable */ const {
+      visits,
+      visit,
+      comments,
+      time,
+      articles,
+      avgArticles,
```

Comment:
I assume using camel case here doesn't break anything?

Reply:
The casing was an issue with the HTML data attributes. It should be fine within Preact.

Reply:
Excellent, thank you for the clarification.